includes/class-comment.php

```diff
@@ -335,4 +346,33 @@ public static function generate_id( $comment ) {
 			\trailingslashit( site_url() )
 		);
 	}
+
+	/**
+	 * Enqueue scripts for remote comments
+	 */
+	public static function enqueue_scripts() {
+		if ( ! is_singular() ) {
+			return;
+		}
+		$handle     = 'activitypub-remote-reply-script';
+		$data       = array(
+			'namespace' => ACTIVITYPUB_REST_NAMESPACE,
+		);
+		$js         = sprintf( 'var _activityPubOptions = %s;', wp_json_encode( $data ) );
+		$asset_file = ACTIVITYPUB_PLUGIN_DIR . 'build/remote-reply/index.asset.php';
+
+		if ( \file_exists( $asset_file ) ) {
+			$assets = require_once $asset_file;
+
+			\wp_enqueue_script(
+				$handle,
+				\plugins_url( 'build/remote-reply/index.js', __DIR__ ),
+				$assets['dependencies'],
+				$assets['version'],
+				true
+			);
+			\wp_enqueue_style( 'wp-components' );
```

Comment:
I missed that we had some custom styles for this, instead of just adding `wp-components`.

So something like

```php
\wp_enqueue_style( 
  'activitypub-remote-reply',
  \plugins_url( 'build/remote-reply/style-index.css', __DIR__ ),
  [ 'wp-components' ],
  $assets['version']
);